src/classes/HiFiAudioAPIData.ts

```diff
@@ -44,6 +44,12 @@ export class HiFiAudioAPIData {
      * A volume level below this value is considered background noise and will be smoothly gated off.
      * The floating point value is specified in dBFS (decibels relative to full scale) with values between -96 dB (indicating no gating)
      * and 0 dB (effectively muting the input from this user). It is in the same decibel units as the VolumeDecibels component of UserDataSubscription.
+     * Setting this value to `NaN` or `null` will cause the volume threshold from the space to be used instead.
+     * 
+     * **COMPATIBILITY WARNING:** In the future, the High Fidelity Audio API server will only fall back to the space volume threshold
+     * if the threshold is `NaN`.
+     *
+     * If you don't supply a `volumeThreshold` when constructing instantiations of this class, it will be ignored.
```

Comment:
I didn't follow what this sentence means, here and below.

For example, does not setting it mean that the default for the space will be used?

Reply:
I understand what you mean. I will try to clarify.